src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/tests/test_iam_cfn_deploy_role_policy.py

```diff
@@ -188,7 +196,10 @@ def test_grant_access_to_kms_keys_new_key_single_resource(iam_client):
     )
     policy_doc_before = deepcopy(instance.policy_document)
 
-    new_key_arn = 'arn:aws:kms:eu-west-1:111111111111:key/new_key'
+
+    test_region = "eu-west-1" if PARTITION == "aws-cn" else "cn-north-1"
```

Comment:
Logic in the tests seems like a red flag. Also "eu-west-1" IF partition == aws-cn seems incorrect